### ISSUE TITLE
Iteration Plan of v2.14.0 - June 2021

### ISSUE THREAD

**Issue by ZwwWayne** [open-mmlab/mmdetection#5279]
MMDetection v2.13.0 has been released [here](https://github.com/open-mmlab/mmdetection/releases/tag/v2.13.0).
We list our plan for v2.14.0, which will be released at the end of June.
The PRs from the communities will also be listed here.
Thus, developers could comprehensively know the plan and the progress.

Algorithms:

- [x] SSD Lite

Refactors:

- [ ] Refactor ONNX export of one-stage detectors #5251

Features:
- [ ] Add dataset downloading and extraction scripts for convenience
- [x] Support PointRend model exporting to ONNX #4977
- [ ] NMS update to speed up exporting to ONNX #4990
- [x] Make YOLOv3 neck more flexible #5218
- [ ] Support to show prediction results without bounding boxes #5275

Enhancements:
- [x] Add `simple_test` to dense heads and unify the mixins for dense heads #5061
- [x] Add documentation for ImageNet pretrained models #5268
- [x] Release RetinaNet, Mask R-CNN, and Faster R-CNN models pre-trained by multi-scale 3x schedule
- [x] Add Chinese documentation #5299

Fixes:

If you have created a PR and wish it to be merged in this version, feel free to leave a message in this issue. We will update our plan accordingly here.

**Notice**: This month we are going through big refactoring on one-stage detectors and documentation. We plan to fix the issues in documentation and the code as much as we can. So please feel free to leave a message here to discuss which part of the documentation needs to be improved. Directly creating a PR is the most appreciated.

**Comment by fcakyon** [comment]
@ZwwWayne how about: Mosaic Augmentation (#4937) ?

**Comment by ZwwWayne** [comment]
> @ZwwWayne how about: Mosaic Augmentation (#4937) ?

We are still working on that. We may support that in July.
